lib/deprecation-cop-view.coffee

```diff
@@ -50,33 +59,73 @@ class DeprecationCopView extends ScrollView
     # afterAttach is called 2x when dep cop is the active pane item on reload.
     return if @subscribedToEvents
 
+    self = this
+
+    @openBlogPost.on 'click', ->
+      require('shell').openExternal('http://blog.atom.io/2015/05/01/removing-deprecated-apis.html')
+      false
+
     @on 'click', '.deprecation-info', ->
       $(this).parent().toggleClass('collapsed')
 
+    @on 'click', '.check-for-update', ->
+      atom.workspace.open('atom://config/updates')
+      false
+
+    @on 'click', '.disable-package', ->
+      if @dataset.packageName
+        atom.packages.disablePackage(@dataset.packageName)
+      false
+
     @on 'click', '.stack-line-location, .source-url', ->
       pathToOpen = @href.replace('file://', '')
       pathToOpen = pathToOpen.replace(/^\//, '') if process.platform is 'win32'
       atom.open(pathsToOpen: [pathToOpen])
 
     @on 'click', '.issue-url', ->
-      # win32 can only handle a 2048 length link, so we use the shortener.
-      return if process.platform isnt 'win32'
-
-      openExternally = (urlToOpen=@href) =>
-        require('shell').openExternal(urlToOpen)
-
-      $.ajax 'http://git.io',
-        type: 'POST'
-        data: url: @href
-        success: (data, status, xhr) ->
-          openExternally(xhr.getResponseHeader('Location'))
-        error: ->
-          openExternally()
-
+      self.openIssueUrl(@dataset.repoUrl, @dataset.issueUrl, @dataset.issueTitle)
       false
 
     @subscribedToEvents = true
 
+  findSimilarIssues: (repoUrl, issueTitle) ->
```

Comment:
I started working on this as well so wanted to mention something I was concerned about.

This function uses GitHub's search API and makes unauthenticated requests. The rate limit for those types of requests is [5 reqs/minute per IP address](https://developer.github.com/v3/search/#rate-limit). I'm concerned that most users who installed packages which use deprecated APIs will have more than 5 deprecations and will easily step over the rate limit here. While that might not be a problem for this package (it will just not return a URL and carry on), it's expected that applications using the API are Good Citizens and [respect the rate limits](https://developer.github.com/guides/best-practices-for-integrators/#dealing-with-rate-limits). This method is not checking the rate limits, and I can imagine scenarios where more than a few requests are made over the limit. I guess the reason why this isn't such a big problem in the notifications package is that exceptions are rare and don't happen all at the same time normally. In deprecation cop, all deprecations are checked at the same time, and anytime the view is opened, I believe.

I wouldn't say this should block this feature from shipping, but I'd :heart: if we could make deprecation cop respect the APIs rate limits (the API returns [rate limit information with every response](https://developer.github.com/v3/#rate-limiting) and provides a separate endpoint for [checking the limits](https://developer.github.com/v3/rate_limit/)) or at least prevent it from making a large number of requests within a short period of time.